x-pack/plugins/security_solution/public/common/containers/sourcerer/index.tsx

```diff
@@ -334,12 +349,14 @@ export const useSourcererDataView = (
 
   const indicesExist = useMemo(
     () =>
-      checkIfIndicesExist({
-        scopeId,
-        signalIndexName,
-        patternList: sourcererDataView.patternList,
-      }),
-    [scopeId, signalIndexName, sourcererDataView]
+      loading || sourcererDataView.loading
```

Comment:
hmm is this what was causing the no data page to flash?

Reply:
I think so